integration_tests/configs/default.jsonnet

```diff
@@ -8,6 +8,10 @@
       mempool: {
         version: 'v1',
       },
+      consensus: {
+        create_empty_blocks: false,
```

Comment:
this is meaningless in cosmos-sdk if I recall correctly.

Reply:
If we fire pending tx events then might need [EnableTxsAvailable](https://github.com/cometbft/cometbft/blob/v0.37.x/node/node.go) which need change for two [configs](https://github.com/cometbft/cometbft/blob/v0.37.x/config/config.go#L1092)